### DISCUSSION TITLE
Using the old renderer

### DISCUSSION THREAD

**Discussion by 9551-Dev** [RS485/LogisticsPipes#1690] [Q&A] [answered]
Hello! i wanted to ask if its possible in any way to get the old pipe models

This is mainly for nostalgic and rendering performance sake

i know that this used to be possible to enable the old renderer in the pipe controller but it is no longer present. I have done search through the code and the commit history to try and find when this was remove and see whether or not i could restore it, however i was not successful.

if anyone knows how this could be done i would really appreciate any help, modifying the source code and building the mod is fine by me

**Comment by bziemons** [comment, marked as answer]
The old models and rendering code were never ported to newer MC/Forge versions (changes in model loading and chunk baking specifically made this time consuming). If you'd like to implement it, you would need to write it from scratch. One way would be to look at BuildCraft code for newer MC versions.

**Reply by 9551-Dev** [reply]
thank you, if i have the free time i will definitely look into it. might as well be a good way to enhance my modding skills
